packages/documentation-v7/src/stories/components/table/table.stories.ts

```diff
@@ -0,0 +1,236 @@
+      description: 'Border style',
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default (underlined)',
+          'table-bordered': 'Bordered',
+          'table-borderless': 'Borderless',
+        },
+      },
+      options: ['null', 'table-bordered', 'table-borderless'],
+      table: {
+        category: 'General',
+      },
+    },
+    variant: {
+      name: 'Variants',
+      description: 'Stylistic table variants.',
+      control: {
+        type: 'check',
+        labels: {
+          'table-striped': 'Striped',
+          'table-hover': 'Hover enabled',
+        },
+      },
+      options: ['table-striped', 'table-hover'],
+      table: {
+        category: 'General',
+      },
+    },
+    alignment: {
+      name: 'Alignment',
+      description:
+        'Vertical alignment options for cell content. Alignment classes can also be set for individual rows or cells.',
+      control: {
+        type: 'select',
+        labels: {
+          'align-top': 'Top (default)',
+          'align-middle': 'Middle',
+          'align-bottom': 'Bottom',
+        },
+      },
+      options: ['align-top', 'align-middle', 'align-bottom'],
+      table: {
+        category: 'General',
+      },
+    },
+  },
+};
+
+export default meta;
+
+type Story = StoryObj;
+
+function renderTable(args: Args) {
+  const cationTop = args.captionPlacement === 'top' ? 'caption-top' : 'null';
+  return html`
+    <table
+      class=${mapClasses({
+        table: true,
+        [args.borderStyle]: args.borderStyle && args.borderStyle !== 'null',
+        [args.variant]: args.variant && args.variant !== 'null',
+        [cationTop]: cationTop !== 'null',
+        [args.alignment]: args.alignment && args.alignment !== 'null',
+      })}
+    >
+      <caption class=${args.captionPlacement === 'hidden' ? 'visually-hidden' : ''}>
+        ${args.caption}
+      </caption>
+      <thead>
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">First</th>
+          <th scope="col">Last</th>
+          <th scope="col">Handle</th>
+        </tr>
+      </thead>
+
+      <tbody>${args.content}</tbody>
+    </table>
+  `;
+}
+
+export const Default: Story = {
+  args: {
+    content: [
+      html`
+        <tr>
+          <th scope="row">1</th>
+          <td>Mark</td>
+          <td>Otto</td>
+          <td>@mdo</td>
+        </tr>
+      `,
+      html`
+        <tr>
+          <th scope="row">2</th>
+          <td>Jacob</td>
+          <td>Thornton</td>
+          <td>@fat</td>
+        </tr>
+      `,
+      html`
+        <tr>
+          <th scope="row">3</th>
+          <td>Larry</td>
+          <td>the Bird</td>
+          <td>@twitter</td>
+        </tr>
+      `,
+      html`
+        <tr>
+          <th scope="row">4</th>
+          <td>Someone</td>
+          <td>
+            else
+            <br />
+            entirely
+          </td>
+          <td>@twitter</td>
+        </tr>
+      `,
+    ],
```

Comment:
With react it was easier to have an array with jsx elements, this is not the case with lit anymore, even if it's not wrong.

```suggestion
    content: html`
      <tr>
        <th scope="row">1</th>
        <td>Mark</td>
        <td>Otto</td>
        <td>@mdo</td>
      </tr>
      <tr>
        <th scope="row">2</th>
        <td>Jacob</td>
        <td>Thornton</td>
        <td>@fat</td>
      </tr>
      <tr>
        <th scope="row">3</th>
        <td>Larry</td>
        <td>the Bird</td>
        <td>@twitter</td>
      </tr>
      <tr>
        <th scope="row">4</th>
        <td>Someone</td>
        <td>
          else
          <br />
          entirely
        </td>
        <td>@twitter</td>
      </tr>
    `,
```